LSP-vscode-stylelint.sublime-settings

```diff
@@ -0,0 +1,46 @@
+{
+	"languages": [
+		{
+			"languageId": "scss",
+			"scopes": ["source.scss"],
+			"syntaxes": ["Packages/Sass/Syntaxes/SCSS.sublime-syntax"],
+		},
+		{
+			"languageId": "css",
+			"scopes": ["source.css", ],
+			"syntaxes": ["Packages/Sass/Syntaxes/CSS.sublime-syntax"],
+		},
+		{
+			"languageId": "vue",
+			"scopes": ["text.html.vue"],
+			"syntaxes": ["Packages/Vue Syntax Highlight/Vue Component.sublime-syntax"],
+		},
```

Comment:
We could look into adding more languages.

Reply:
We can probably add a guide in the README on how to activate/add stylelint for any kind of file.